### ISSUE TITLE
Rejuvanation and Anti-Grav explosives and missiles Broken 1.16.5

### ISSUE THREAD

**Issue by Sp0okz98** [jdawg3636/ICBM#7]
We have been attempting to get the Rejuvanation and Anti gravity explosives to work in this pack but they just do nothing, they make the sound of lit tnt, make the audible sounds, but no effects, no explosions, nothing.

**Comment by jdawg3636** [comment]
Yep! This is known; many features in the mod are not yet implemented, including those two explosives. They will be implemented in a future release. I'll leave this issue open until they're added. Check out my Discord for more frequent updates! https://discord.gg/7wZzJTC
